integrationtests/src/main/java/io/kroxylicious/proxy/testkafkacluster/KafkaClusterConfig.java

```diff
@@ -138,6 +136,23 @@ public Stream<ConfigHolder> getBrokerConfigs(Supplier<KafkaEndpoints> endPointCo
                 server.put(String.format("listener.name.%s.plain.sasl.jaas.config", "EXTERNAL".toLowerCase()), plainModuleConfig);
             }
 
+            if (securityProtocol != null && securityProtocol.contains("SSL")) {
+                KeytoolCertificateGenerator keytoolCertificateGenerator = new KeytoolCertificateGenerator();
+                try {
+                    keytoolCertificateGenerator.generateSelfSignedCertificateEntry("test@redhat.com", clientEndpoint.getConnect().getHost(), "KI", "RedHat", null, null,
```

Comment:
Should we have way of injecting proper certificates here? Or does self-signed cover all the use cases we want to support/test?

cc: @k-wall

Reply:
TBH I don't really know. If needed we could create a new ticket to include the chance to use an external certificate and import it to be used by client and server.

Reply:
We'll probably need something more sophisticated  at some point, but I think this is OK for now.